addons/crm/models/crm_lead.py

```diff
@@ -1231,14 +1231,15 @@ def get_empty_list_help(self, help):
             help_title = _('Create a new lead')
         else:
             help_title = _('Create an opportunity to start playing with your pipeline.')
-        alias_record = self.env['mail.alias'].search([
+        sales_team = self.env.user.sale_team_id
+        alias_record = sales_team.alias_id if sales_team.active else self.env['mail.alias'].search([
             ('alias_name', '!=', False),
             ('alias_name', '!=', ''),
             ('alias_model_id.model', '=', 'crm.lead'),
             ('alias_parent_model_id.model', '=', 'crm.team'),
-            ('alias_force_thread_id', '=', False)
+            ('alias_force_thread_id', '=', False),
         ], limit=1)
-        if alias_record and alias_record.alias_domain and alias_record.alias_name:
+        if all([alias_record.alias_domain, alias_record.alias_name]):
```

Comment:
I don't think this changes anything functionally? In that case best to leave it as is, for blaming purposes notably

Same with the comma above